index.js

```diff
@@ -5,7 +5,9 @@
  */
 var boot = require( 'boot' ),
 	http = require( 'http' ),
-	chalk = require( 'chalk' );
+	https = require( 'https' ),
```

Comment:
might as well use `http2` here? https://nodejs.org/api/http2.html#http2_server_side_example

Reply:
Do you know if `http2` is compatible with `http` or `https`?

Reply:
It seems to be compatible 😃.

Reply:
… in appearance only. I've errors when using the `http2` package. I would stick with `https` for now, and see if we need `http2` later, that's not my priority for now :smiley:.